Dockerfile.rhel7

```diff
@@ -1,17 +1,11 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
 WORKDIR /go/src/github.com/openshift/cluster-authentication-operator
 COPY . .
-RUN go build -tags="ocp" -o authentication-operator ./cmd/authentication-operator
+ENV GO_PACKAGE github.com/openshift/cluster-authentication-operator
+RUN go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit=$(git describe --long --tags --abbrev=7 --match 'v[0-9]*')" -tags="ocp" -o authentication-operator ./cmd/authentication-operator
 
 FROM registry.svc.ci.openshift.org/ocp/4.2:base
 COPY --from=builder /go/src/github.com/openshift/cluster-authentication-operator/authentication-operator /usr/bin/
 COPY manifests /manifests
 ENTRYPOINT ["/usr/bin/authentication-operator"]
-LABEL io.k8s.display-name="OpenShift cluster-authentication-operator" \
-      io.k8s.description="This is a component of OpenShift and manages cluster authentication settings" \
-      com.redhat.component="cluster-authentication-operator" \
-      maintainer="OpenShift Auth Team <aos-auth-team@redhat.com>" \
-      name="openshift/ose-cluster-authentication-operator" \
-      version="v4.0.0" \
-      io.openshift.tags="openshift" \
-      io.openshift.release.operator=true
+LABEL io.openshift.release.operator=true
```

Comment:
with any changes to the file, need to make sure it will not disrupt/break anything with ART